### ISSUE TITLE
Saw you are using Tika and Elasticsearch

### ISSUE THREAD

**Issue by chrismattmann** [vlall/darksearch#6]
thanks that's awesome! thanks for using tika-python!

**Comment by vlall** [comment]
Absolutely, it's extremely helpful! I'm hoping to improve Elastic Search capabilities, and maybe even integrate Apache Nutch for crawling.

**Comment by chrismattmann** [comment]
Awesome if you integrate Nutch check out http://github.com/chrismattmann/nutch-python.git :) :+1:
